test/modules/altitudeProfile/utils/altitudeProfileUtils.test.js

```diff
@@ -0,0 +1,19 @@
+import { $injector } from '../../../../src/injection';
+import { SlopeType } from '../../../../src/modules/altitudeProfile/utils/altitudeProfileUtils';
+
+$injector.registerSingleton('TranslationService', { translate: (key) => key });
+
+describe('Unit test functions altitudeProfileUtils.js', () => {
+	describe('check SlopeType', () => {
+		const slopeType = SlopeType.Flat;
+
+		it('check SlopeType length', () => {
+			expect(Object.keys(slopeType).length).toBe(4);
+		});
```

Comment:
Tests the number of characters of "Flat", don`t know if it's the test you've wanted 😉